packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -305,14 +313,11 @@ export default {
 .spaces-list {
   &-card {
     box-shadow: none !important;
+    border-radius: 3px;
```

Comment:
There is an `oc-rounded` css class from the design system. Could you try to use that instead of applying the 3px border radius here and in line 335ff?